### PR TITLE
fixed tests in FindPositionsCommandTests

### PR DESCRIPTION
**Pull request by VLiamin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/636

### CODE REVIEW

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+            _findResultResponse = new()
+            {
+                Status = OperationResultStatusType.FullSuccess,
+                Body = _positionsList,
+                TotalCount = 1
+            };
+
+            /*            _mapperMock
```
Comment (BelehovEgor) [inline test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:66]: delete

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenRepositoryThrowsException()
+        {
+            int value;
+            _autoMock
+                .Setup<IPositionRepository, List<DbPosition>>(x => x.Find(0, 15, true, out value))
+                .Throws(new Exception());
+            Assert.That(() => _command.Execute(0, 15, true), Throws.TypeOf<Exception>());
```
Comment (BelehovEgor) [inline test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:87]: you can use Assert.Throws<TypeOfException>

**Review by BelehovEgor — APPROVED** [review]

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs (hunk 1 above)
Comment (VLiamin) [reply test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs]: fixed

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs (hunk 2 above)
Comment (VLiamin) [reply test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs]: fixed

**Review by Cheephot — APPROVED** [review]

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                .Returns(_dbPositionsList);
+
+            _autoMock
+                .Setup<IPositionInfoMapper, PositionInfo>(x => x.Map(dbPosition))
+                .Returns(position);
+
+            _command = new FindPositionsCommand(_autoMock.GetMock<IPositionRepository>().Object,
+                _autoMock.GetMock<IPositionInfoMapper>().Object);
+
+
```
Comment (MariyaSin) [inline test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:58]: remove empty line

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            _findResultResponse = new()
+            {
+                Status = OperationResultStatusType.FullSuccess,
+                Body = _positionsList,
+                TotalCount = 1
```
Comment (MariyaSin) [inline test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:63]: you create a variable "value", why did you use it here?

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenRepositoryThrowsException()
+        {
+            int value;
+            _autoMock
+                .Setup<IPositionRepository, List<DbPosition>>(x => x.Find(0, 15, true, out value))
+                .Throws(new Exception());
+            Assert.Throws<Exception>(() => _command.Execute(0, 15, true));
```
Comment (MariyaSin) [inline test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:83]: add check what mapper Times.Never() and repository Times.Once()

Comment (VLiamin) [reply test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs:82]: done

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs (hunk 4 above)
Comment (VLiamin) [reply test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs]: done

File: test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs (hunk 3 above)
Comment (VLiamin) [reply test/CompanyService.Business.UnitTests/Commands/Position/FindPositionsCommandTests.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**